### PR TITLE
CarpetX: Move unused variables inside CCTK_DEBUG

### PR DESCRIPTION
**Pull request by lwJi**

### CODE REVIEW

**Review by rhaas80 — APPROVED** [review]

**Comment by rhaas80** [comment]
Needs to fix conflicts before it can be merged in.

**Comment by lwJi** [comment]
> Needs to fix conflicts before it can be merged in.

Done. But I think the CI is broken for the moment

**Comment by eschnett** [comment]
I'm very close to merging the CI fix.

**Comment by lwJi** [comment]
> I'm very close to merging the CI fix.

Thanks, @eschnett

File: CarpetX/src/prolongate_3d_rf2_impl.hxx [hunk 1]
```diff
@@ -1339,14 +1339,14 @@ void prolongate_3d_rf2<
 #endif
       return crseptr[crsebox.index(vcrse)];
     };
+#ifdef CCTK_DEBUG
```
Comment (eschnett) [inline CarpetX/src/prolongate_3d_rf2_impl.hxx:1342]: Same here.

**Review by eschnett — CHANGES_REQUESTED** [review]
I would prefer a `CCTK_ATTRIBUTE_MAYBE_UNUSED` instead.

**Pull request closed without merging**